yogabprawira/fibonacci.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func fibo(a int, b int) int {
+	c := a + b
+	fmt.Print(c, " ")
+	if c > 300 {
```

Comment:
kalo untuk code asli sebisa mgkin limit nya dalam bntuk variable, kalo gini kan ga gelas 300 itu value apa dan buat apa, atau bisa d kasi komen penjelasan d atas nya

Reply:
got it